### PR TITLE
Added Guild logos and few minor fixes

### PR DESCRIPTION
**Pull request by ihardzeenka**

### CODE REVIEW

File: content/docs/role/_index.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 weight: 500
 title: "Role"
 description: "Definition of the role, key responsibilities and types of Software Architects "
-icon: "design_services"
+icon: "person"
```
Comment (DenisKlimenko) [inline content/docs/role/_index.md:5]:
```suggestion
icon: "badge"
```

Looks more suitable for me

Comment (ihardzeenka) [reply content/docs/role/_index.md:5]: I do not have a preference. What do you think @gekola ?

Comment (gekola) [reply content/docs/role/_index.md:5]: `"badge"` seems better to me.

File: themes/lotusdocs/assets/scss/custom/pages/_hero.scss [hunk 2]
```diff
@@ -12,4 +12,6 @@
 .hero {
     background-size: cover;
     padding-top: 70px;
+    min-height: 800px;
```
Comment (gekola) [inline themes/lotusdocs/assets/scss/custom/pages/_hero.scss:15]: Bad for mobile clients. But we'll replace it eventually anyway.

**Review by gekola — APPROVED** [review]

File: content/docs/role/_index.md (hunk 1 above)
Comment (ihardzeenka) [reply content/docs/role/_index.md]: changed

**Review by DenisKlimenko — APPROVED** [review]

**Pull request merged**